vision-annotate-images/README.md

```diff
@@ -0,0 +1,16 @@
+# Vision Annotate Images
+
+This function acts as a pass through to the [Cloud Vision JSON API](https://cloud.google.com/vision/docs/request). 
+
+To call this from an app, you must first sign-in using [Firebase Auth](https://firebase.google.com/docs/auth).
+Only users who pass a valid Firebase ID token as a Bearer token in the `Authorization` header of the HTTP request are authorized to use the function.
+
```

Comment:
I'm thinking once the firebase docs are ready we can just link to those as an example of how to use this in an app. What do you think?

Reply:
Yep sg